#!/usr/bin/env python
PACKAGE = "mitro_kinect"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("voxel_size",        double_t, 0, "Voxel size!",           0.025, 0.01,  0.2)
gen.add("plane_tresh",       double_t, 0, "Plane treshold",        0.03,  0.005, 0.1)
gen.add("outlier_neighbors", int_t,    0, "Outlier min neighbors", 10,    1,     100)
gen.add("outlier_radius",    double_t, 0, "Outlier radius",        0.1,   0.01,  0.2)

exit(gen.generate(PACKAGE, "mitro_kinect", "MitroKinect"))
